src/main/webapp/WEB-INF/views/LandingPage.jsp

```diff
@@ -0,0 +1,105 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
+  <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+    <!DOCTYPE html>
+    <html lang="en">
+
+    <head>
+      <meta charset="UTF-8">
```

Comment:
nice consistent header here, you got the charset, lang=en, and consistency with the jsp encoding declaration